exporter/metric/option.go

```diff
@@ -116,3 +119,11 @@ func DefaultResourceAttributesFilter(kv attribute.KeyValue) bool {
 func NoAttributes(attribute.KeyValue) bool {
 	return false
 }
+
+// WithDisableCreateMetricDescriptors will disable the automatic creation of
+// MetricDescriptors when an unknown metric is set to be exported.
+func WithDisableCreateMetricDescriptors(create bool) func(o *options) {
```

Comment:
This should just take no arguments and set the field to true. By default it's false, so if they don't call this option then the default setup makes sense.

Reply:
Changed in: [33257b3](https://github.com/avilevy18/opentelemetry-operations-go/pull/1/commits/33257b31e3a9cec0824a4a00369e9f7a587ab8a8)